kernel/packages/unity-interface/BrowserInterface.ts

```diff
@@ -377,6 +378,45 @@ export class BrowserInterface {
     globalThis.globalStore.dispatch(updateFriendship(action, userId.toLowerCase(), false))
   }
 
+  public SearchENSOwner(data: { name: string; maxResults: number }) {
```

Comment:
Could we set up a max here? What default value are you using right now?

Reply:
I made `maxResults` as optional